test/models/concerns/smart_proxy_extensions_test.rb

```diff
@@ -414,7 +414,7 @@ def test_sync_container_gateway
       capsule_content.smart_proxy.add_lifecycle_environment(environment)
 
       expected_repo_list_args = {
-        :repositories => [{:repository => "empty_organization-puppet_product-busybox", :auth_required => true}, {:repository => "busybox", :auth_required => true}],
+        :repositories => [{:repository => "empty_organization-puppet_product-busybox", :auth_required => true}, {:repository => "busybox", :auth_required => true}, {:repository => "id/1/2/container-push-repo", :auth_required => true}],
```

Comment:
I had to amend several existing tests like this, because I added another container type fixture repo, and these tests make assertions about "the list of all container type repos" in the fixtures.